python-threatexchange/threatexchange/exchanges/storage.py

```diff
@@ -0,0 +1,81 @@
+import dbm
+from abc import ABC, abstractmethod
+from threatexchange.exchanges.collab_config import CollaborationConfigBase
+from typing import TypeVar 
+import typing as t
+import pickle
+
+T = t.TypeVar('T')
+
+class GenericStorage(ABC):
+
+    @abstractmethod
+    def connect(self, *args, **kwargs):
```

Comment:
If we require that a constructing an instance of this object make it functional, do we need this method?